apps/mobile/src/screens/Dashboard/Profile/index.tsx

```diff
@@ -1,22 +1,30 @@
-import type { FC } from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-import type { StackScreenProps } from '@react-navigation/stack';
-import type { DashboardParamList } from 'utils/navigation';
+import { View } from 'react-native';
+import { useSafeAreaInsets } from '@walless/app';
+import { ProfileFeature } from '@walless/app/features/Profile';
+import { navigate } from 'utils/navigation';
 
-type Props = StackScreenProps<DashboardParamList, 'Profile'>;
+export const ProfileScreen = () => {
+	const { top } = useSafeAreaInsets();
+
+	const containerStyle = {
+		paddingTop: top,
+		flex: 1,
+	};
+
+	const handleSettingPress = () => {
+		navigate('Dashboard', {
+			screen: 'Profile',
+			params: {
+				screen: 'Setting',
+			},
+		});
+	};
 
-export const ProfileScreen: FC<Props> = () => {
 	return (
-		<View style={styles.container}>
-			<Text>ProfileScreen</Text>
+		<View style={containerStyle}>
```

Comment:
in this case, we don't need to wrap `ProfileFeature` under `View` use `<ProfileFeature style={...} />` instead